src/Microsoft.Health.Fhir.ValueSets/FhirAnonymousOperationType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+namespace Microsoft.Health.Fhir.ValueSets
+{
+    /// <summary>
+    /// Value set for Fhir operations which do not require authorization
+    /// </summary>
+    public static class FhirAnonymousOperationType
+    {
+        public const string Metadata = "metadata";
+
+        public const string Versions = "versions";
```

Comment:
Should the health check endpoint be included here?

Reply:
health check call is wired a little differently maybe we can do that in a separate story